front-end/src/Components/Layouts/EditOrganization.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  Button,
+  Card,
+  Container,
+  Grid,
+  Typography,
+  ListItem,
+  ListItemAvatar,
+  Avatar,
+  ListItemText,
+  Fab,
+  makeStyles,
+  Dialog,
+  AppBar,
+  Toolbar,
+  IconButton,
+  List,
+  Divider,
+} from "@material-ui/core";
+import React, { Fragment } from "react";
+import { NotificationMessage } from "../Misc/Notifications";
+import EditIcon from "@material-ui/icons/Edit";
+import classes from "*.module.css";
+import CloseIcon from "@material-ui/icons/Close";
+import ViewEditOrganizationDescription from "../Content/OrganizationInformation/ViewEditOrganizationDescription";
+import ViewEditOrganizationEmail from "../Content/OrganizationInformation/ViewEditOrganizationEmail";
+import ViewEditOrganizationPhone from "../Content/OrganizationInformation/ViewEditOrganizationPhone";
+import ViewEditOrganizationPhoto from "../Content/OrganizationInformation/ViewEditOrganizationPhoto";
+import ViewEditOrganizationName from "../Content/OrganizationInformation/ViewEditOrganizationName";
+interface EventInfoProps {
+  setNotification: (notification: NotificationMessage) => void;
+  currentOrganizationData: any;
+  setCurrentOrganizationData: any;
+  setLoadingMessage: any;
+  handleLoadOrganizationData: any;
+  classes: any;
+}
+
+const Organization: React.FunctionComponent<EventInfoProps> = ({
```

Comment:
Can we call this "EditOrganization" to match the file name?

Reply:
Done